File: z-clients/javascript/src/models/sinkConfigureOut.ts
// this file is @generated

export interface SinkConfigureOut {
    topic: string;
    consumerGroup: string;
}

export const SinkConfigureOutSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): SinkConfigureOut {
        return {
            topic: object['topic'],
            consumerGroup: object['consumer_group'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: SinkConfigureOut): any {
        return {
            'topic': self.topic,
            'consumer_group': self.consumerGroup,
        };
    }
}

File: z-clients/rust/src/models/sink_configure_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct SinkConfigureOut {
    pub topic: String,

    pub consumer_group: String,
}

impl SinkConfigureOut {
    pub fn new(topic: String, consumer_group: String) -> Self {
        Self {
            topic,
            consumer_group,
        }
    }
}
